cargo-sqlx/.gitignore

```diff
@@ -1,4 +1,6 @@
 /target
 /migrations
 Cargo.lock
-.env
\ No newline at end of file
+.env
+.env-pg
+**/*.sql
```

Comment:
Did you commit this by mistake? I don't think we want to forbid having _any_ `.sql` files in this part of the project, and `.env-pg` is out of place.

Reply:
Yeah, accidentally ignored some of my dev files.